svixhq/svix-libs
# php: initial verification lib

Again not very familiar with PHP but added phpunit tests to try and cover our bases.

To test you can run:
```sh
composer install
./vendor/bin/phpunit tests
```
We will probably want to add this to https://packagist.org

Example Usage:
```php
$secret = 'MfKQ9r8GKYqrTwjUPD8ILPZIo2LaLaSw';
$payload = '{"test": 2432232314}';
$header = array(
        'svix-id'  => 'msg_p5jXN8AQM9LWM0D4loKWxJek',
        'svix-timestamp' => '1614265330',
        'svix-signature' => 'v1,g0hM9SsE+OTPJTGt/tmIKtSyZlE3uFJELVlNIOLJ1OE=',
    );

$wh = new \Svix\Webhook($secret);
$json = $wh->verify($payload, $header);
```

Closes #30
